doc/development/source/tutorial-settings.rst

```diff
@@ -130,6 +130,31 @@ Well-behaved widgets remember their settings - the state of their
 checkboxes and radio-buttons, the text in their line edits, the
 selections in their combo boxes and similar.
 
+Persisting defaults
+*******************
+Default setting values are not set in stone. As different users might
+prefer different defaults, Orange remembers which settings were used
+and stores them as the new defaults when the widget is deleted. Widgets
+removed from there on will start with the new defaults.
```

Comment:
removed -> created

Or replace the paragraph with a shorter *When a widget is removed, its settings are stored to be used as defaults for future instances of this widget.*